server/config/config.go

```diff
@@ -129,6 +138,10 @@ func DefaultConfig() *Config {
 			RPCReadTimeout:     10,
 			RPCMaxBodyBytes:    1000000,
 		},
+		GRPC: GRPCConfig{
```

Comment:
IMHO it's going to be very weird to operators to have both an API server and a gRPC server. Is there no way we can server requests from a single point?

Reply:
I think it'd be more feasible to run grpc-gateway on the same HTTP server: https://github.com/grpc-ecosystem/grpc-gateway#usage.

But then we would need to be more careful with path names. I had the idea that the `google.api.http` path names would be the same as the current REST paths, although maybe we do something different so they don't conflict.

Reply:
Also I don't actually think having similar, but different services on different ports is that weird...

Reply:
I'm indifferent to the ultimate solution, as long as we can:

1. mount arbitrary HTTP handlers to the router
2. have a single interface

Reply:
I Agree, managing 2 servers is tough for an end user. But gRPC is something which can/should stand alone as it's 
 easy to maintain it separately. I don't see any other issues than some UX for the enduser and I think that is okay.